ui/raidboss/data/04-sb/raid/o12n.js

```diff
@@ -55,10 +55,10 @@
     },
     {
       id: 'O12N Optimized Blade Dance',
-      regex: / 14:(?:3321|3322):(?:Omega|Omega-M) starts using (?:Unknown_3321|Unknown_3322|Optimized Blade Dance) on (\y{Name})/,
-      regexDe: / 14:(?:3321|3322):(?:Omega|Omega-M) starts using (?:Unknown_3321|Unknown_3322|Omega-Schwertertanz) on (\y{Name})/,
-      regexFr: / 14:(?:3321|3322):(?:Oméga|Oméga-M) starts using (?:Unknown_3321|Unknown_3322|Danse De La Lame Oméga) on (\y{Name})/,
-      regexJa: / 14:(?:3321|3322):(?:オメガ|オメガM) starts using (?:Unknown_3321|Unknown_3322|ブレードダンス・オメガ) on (\y{Name})/,
+      regex: / 14:(?:3321|3322):(?:Omega|Omega-M) starts using Optimized Bladedance/,
+      regexDe: / 14:(?:3321|3322):(?:Omega|Omega-M) starts using Omega-Schwertertanz/,
+      regexFr: / 14:(?:3321|3322):(?:Oméga|Oméga-M) starts using Danse De La Lame Oméga/,
+      regexJa: / 14:(?:3321|3322):(?:オメガ|オメガM) starts using ブレードダンス・オメガ/,
```

Comment:
These lines lost the `on (\y{Name})` capturing group.

Reply:
done :)